test/e2e/policies.test-suite/balloons/n4c16/test10-health-checking/code.var.sh

```diff
@@ -0,0 +1,19 @@
+# Test that
+# Syntatically incorrect configuration file is resulting in the nri-resmgr
+# pod readiness probe failure.
+host-command "$SCP \"${SRC_DIR}/test/e2e/policies.test-suite/balloons/n4c16/test10-health-checking/nri-resmgr-configmap.yaml\" $VM_HOSTNAME:" || {
+    command-error "copying nri-resmgr-configmap.yaml to VM failed"
+}
```

Comment:
This should be replaced by `vm-put-file` as it hides the paths and other nastiness from the user.
See for example `policies.test-suite/balloons/n4c16/test06-update-configmap/code.var.sh` for usage example.